hilti/runtime/src/init.cc

```diff
@@ -69,6 +73,8 @@ void hilti::rt::done() {
                          stats.memory_heap);
     }
 
+    profiler::detail::done();
```

Comment:
Actually `profiler::done` has a check for `profiling_enabled`, not `config.enable_profiling`. `profiling_enabled` is set by `profiler::init`. So the idea is that `rt::init` checks the config option and enables profiling if desired; everything else triggers on whether that has happened. Probably unnecessary to have both, but global state is a bit quicker to access and in the unlikely event that the config option changed, this would still do the right thing.